docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md

```diff
@@ -602,7 +602,60 @@ to the child entities based on the newly created parent entity's ID.
 
 {% endinfo_block %}
 
-#### Error codes
```

Comment:
https://github.com/spryker/suite-nonsplit/pull/8817/files glossary changes should be added to integration guides

Reply:
IG was updated as part of another PR for DELETE https://github.com/spryker/spryker-docs/pull/2686